content/working_with/console/pages/system-health-page.md

```diff
@@ -0,0 +1,19 @@
+---
+layout: bt_wiki
+title: System Health Page
+category: Console
+draft: false
+abstract:
+weight: 180
+aliases: ["/manager_webui/system-health/", "/working_with/console/system-health-page/"]
+---
+
+The page contains [Cluster status widget]({{< relref "working_with/console/widgets/highAvailability.md" >}}).
+
+{{% note %}}
+You can access the System Health page only if you are an admin user.

```

Comment:
Optional:

In other pages the following format is used:
```suggestion
You can access the System Health page only if you are an `admin` user.
```
so maybe let's try to keep it consistent.